source/Plugins/ExpressionParser/Swift/SwiftASTManipulator.cpp

```diff
@@ -837,9 +837,10 @@ void SwiftASTManipulator::MakeDeclarationsPublic() {
       if (swift::ValueDecl *value_decl =
               llvm::dyn_cast<swift::ValueDecl>(decl)) {
         auto access = swift::AccessLevel::Public;
-        if (swift::isa<swift::ClassDecl>(value_decl) ||
-            swift::isa<swift::ClassDecl>(value_decl->getDeclContext())) {
-          access = swift::AccessLevel::Open;
+        if (llvm::isa<swift::ClassDecl>(value_decl) ||
```

Comment:
Can you add a comment here? The surrounding isn't really much commented but we should try improving this situation moving forward.